spec/tic_tac_toe_spec.rb

```diff
@@ -1,14 +1,75 @@
 # frozen_string_literal: true
 
-# require 'bin/main'
-require './lib/player'
+load File.expand_path('../bin/main', __dir__)
+require File.expand_path('../lib/player', __dir__)
+require File.expand_path('../lib/board', __dir__)
 
 RSpec.describe do
-  describe Player do
-    describe '#player_signs' do
-      it "Check if the player sign is 'X' or 'O':" do
-        allow($stdin).to receive(:gets).and_return('X' || 'O')
-      end
+  describe 'board_show' do
+    let(:board_test) { Board.new }
+    it 'shows the board' do
+      expect { board_show(board_test) }.to output(
+        "+---+---+---+\n"\
```

Comment:
It's best practice to be clear about what method you are describing. For instance, use the Ruby documentation convention of `.` (or `::`) when referring to a class method's name and `#` when referring to an instance method's name.
```
describe '.authenticate' do
describe '#admin?' do
```